.gitignore

```diff
@@ -187,6 +187,7 @@ dist/
 coverage.xml
 
 # Jupyter Notebook
+notebooks/
```

Comment:
wait why do we need to ignore a notebooks/ directory at the repo root?

Reply:
I'll remove this. That was an intermediate state.